src/Cake.Jira/Cake.Jira.csproj

```
@@ -1,20 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <AssemblyTitle>Cake.Jira</AssemblyTitle>
     <Company>Hugo Tavares</Company>
     <Product>Cake.Jira</Product>
     <Description>Cake build addin for integration with Jira.</Description>
-    <Copyright>Copyright © Hugo Tavares 2018</Copyright>
+    <Copyright>Copyright © Hugo Tavares 2021</Copyright>
     <Version>0.6.0</Version>
     <AssemblyVersion>0.6.0.0</AssemblyVersion>
     <FileVersion>0.6.0.0</FileVersion>
     <OutputPath>bin\$(Configuration)\</OutputPath>
     <Authors>Hugo Tavares</Authors>
     <PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/cake-contrib-medium.png</PackageIconUrl>
     <PackageId>Cake.Jira</PackageId>
-    <PackageLicenseUrl>https://github.com/Ninglin/Cake.Jira/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/Ninglin/Cake.Jira/</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/cake-contrib/Cake.Jira/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/cake-contrib/Cake.Jira/</PackageProjectUrl>
     <PackageTags>Cake Script Build Jira </PackageTags>
```

```suggestion
    <PackageTags>Cake Script Build Jira cake-addin</PackageTags>
```

In case you want to close https://github.com/cake-contrib/Cake.Jira/issues/15 as part of this PR as well

---

Thanks. Was going to do it next :)